src/Core/MainLoop.js

```diff
@@ -92,8 +92,26 @@ MainLoop.prototype._step = function _step(view, timestamp) {
 
     // update camera
     const dim = this.gfxEngine.getWindowSize();
+    const foo = (dim.width != view.camera.width || dim.height != view.camera.height);
+
+    const animator = {
+        t: 0,
+        update: (dt, ignore) => {
+            this.t = ignore ? 0 : Math.min(1, dt / 1000 + this.t);
```

Comment:
What about setting `this.t = 0` at the same time we call `view.controls.updateCamera` and testing `Date.now() - this.t > 1000` on line 102? I find it easier to read.

Reply:
I agree, let's do that ! ;)